### PR TITLE
Matching engine text-to-image: Added explicit image detection

### PR DESCRIPTION
**Pull request by ivanmkc**
Added explicit image detection using google-cloud-vision safe search API.

### CODE REVIEW

File: notebooks/official/matching_engine/sdk_matching_engine_create_text_to_image_embeddings.ipynb [hunk 1]
```diff
@@ -129,7 +129,7 @@
     {
```
Comment (andrewferlitsch) [inline notebooks/official/matching_engine/sdk_matching_engine_create_text_to_image_embeddings.ipynb:129]: <p>can we combine the installs into a single install cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1624/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (andrewferlitsch) [inline notebooks/official/matching_engine/sdk_matching_engine_create_text_to_image_embeddings.ipynb:129]: <p>See details at -&gt; Learn more about</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1624/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (andrewferlitsch) [inline notebooks/official/matching_engine/sdk_matching_engine_create_text_to_image_embeddings.ipynb:129]: <p>Doesn't the """ ... """ (help string) belong inside the function definition</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1624/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Comment (andrewferlitsch) [inline notebooks/official/matching_engine/sdk_matching_engine_create_text_to_image_embeddings.ipynb:129]: <p>Should have heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1624/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (andrewferlitsch) [inline notebooks/official/matching_engine/sdk_matching_engine_create_text_to_image_embeddings.ipynb:129]: <p>Google Cloud Vision -&gt; Cloud Vision</p><p> </p><p>See .. -&gt; Learn more about</p><p> </p><p>Should have heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1624/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (ivanmkc) [reply notebooks/official/matching_engine/sdk_matching_engine_create_text_to_image_embeddings.ipynb:129]: <p>Done.</p>

Comment (ivanmkc) [reply notebooks/official/matching_engine/sdk_matching_engine_create_text_to_image_embeddings.ipynb:129]: <p>Done.</p>

Comment (ivanmkc) [reply notebooks/official/matching_engine/sdk_matching_engine_create_text_to_image_embeddings.ipynb:129]: <p>Done</p>

Comment (ivanmkc) [reply notebooks/official/matching_engine/sdk_matching_engine_create_text_to_image_embeddings.ipynb:129]: <p>Done.</p>

Comment (sarahcdugan) [inline notebooks/official/matching_engine/sdk_matching_engine_create_text_to_image_embeddings.ipynb:129]: <p>Nit - add a period </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1624/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

**Review by sarahcdugan — APPROVED** [review]

File: notebooks/official/matching_engine/sdk_matching_engine_create_text_to_image_embeddings.ipynb (hunk 1 above)
Comment (ivanmkc) [reply notebooks/official/matching_engine/sdk_matching_engine_create_text_to_image_embeddings.ipynb:129]: <p>Done.</p>

Comment (ivanmkc) [reply notebooks/official/matching_engine/sdk_matching_engine_create_text_to_image_embeddings.ipynb:129]: <p>Done.</p>

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**